content/4_Gold/Divis.mdx

```diff
@@ -168,85 +168,59 @@ for each $x$, find the number of divisors of $x$ in $\mathcal{O}(\sqrt x)$ time.
 <CPPSection>
 
 ```cpp
-#include <bits/stdc++.h>
+#include <iostream>
+
 using namespace std;
 
 int main() {
-	ios_base::sync_with_stdio(0);
-	cin.tie(0);
 	int n;
 	cin >> n;
-	while (n--) {
-		int x, ans = 0;
+	for (int i = 0; i < n; i++) {
```

Comment:
this is probably bad since there's another `i` inside the loop; the while loop is fine, or you should change it to a different variable